datamonster_api/lib/datasource.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, _id, name, category, uri, dm):
         self.uri = uri
         self.dm = dm
 
+    def __eq__(self, obj):
+        return isinstance(obj, Datasource) and self.get_details() == obj.get_details()
```

Comment:
Does it make sense to check equality against the ids (data fountain uuids) instead?

Reply:
updated